arangod/Graph/PathManagement/PathValidatorTracer.cpp

```diff
@@ -46,8 +46,9 @@ using namespace arangodb::graph;
 
 template<class PathValidatorImplementation>
 PathValidatorTracer<PathValidatorImplementation>::PathValidatorTracer(
-    Provider& provider, PathStore& store, PathValidatorOptions opts)
-    : _impl{provider, store, std::move(opts)} {}
+    Provider& provider, PathStore& store, PathValidatorOptions opts,
```

Comment:
I do not think any change in this file should be necessary, the handed in booleans are constant for each traversal, and do not differ for every call of this method.

Reply:
All changes here rolled back.